src/driver.rs

```diff
@@ -82,6 +84,40 @@ pub fn parse_to<
     tok.unwrap().unwrap()
 }
 
+/// Parse an HTML fragment and send results to a `TreeSink`.
+///
+/// ## Example
+///
+/// ```ignore
+/// let mut sink = MySink;
+/// parse_fragment_to(&mut sink, one_input(my_str), context_token, Default::default());
+/// ```
+pub fn parse_fragment_to<
+        Sink: TreeSink,
+        It: Iterator<Item=String>
+    >(
+        sink: Sink,
+        input: It,
+        context: String,
```

Comment:
I think this should be `Atom`, to match the tree builder APIs that talk about node names.  Also, that will avoid a heap allocation in the common case.